zilencer/models.py

```diff
@@ -147,6 +147,18 @@ def __str__(self) -> str:
         return f"{self.host} {str(self.uuid)[0:12]}"
 
 
+class RemoteRealmBillingUser(models.Model):
+    remote_realm = models.ForeignKey(RemoteRealm, on_delete=models.CASCADE)
+
+    # The .uuid of the UserProfile on the remote server
+    user_uuid = models.UUIDField()
+    full_name = models.TextField(default="")
+    email = models.EmailField()
+
+    TOS_VERSION_BEFORE_FIRST_LOGIN = UserProfile.TOS_VERSION_BEFORE_FIRST_LOGIN
+    tos_version = models.TextField(default=TOS_VERSION_BEFORE_FIRST_LOGIN)
```

Comment:
Thinking about whether to use `None` default here or this, looking  at how it's explained on `UserProfile`:
```
# Terms of Service version number that this user has accepted. We
# use the special value TOS_VERSION_BEFORE_FIRST_LOGIN for users
# whose account was created without direct user interaction (via
# the API or a data import), and null for users whose account is
# fully created on servers that do not have a configured ToS.
TOS_VERSION_BEFORE_FIRST_LOGIN = "-1"
tos_version = models.CharField(null=True, max_length=10)
```

 So I figured  one would expect meaning of values to be somewhat consistent across “tos_version”-like  attributes on different models. `None` doesn’t make sense in the context of the new model, since it’s zilencer-only so there's always ToS. Therefore I went with TOS_VERSION_BEFORE_FIRST_LOGIN default.

Reply:
Yeah that seems reasonable I guess.